Slack.Automation/Promact.Core.Repository/ScrumReportRepository/ScrumReportRepository.cs

```diff
@@ -90,22 +90,32 @@ public ScrumReportRepository(IRepository<Scrum> scrumDataRepository, IRepository
             }
             foreach (var todayScrumAnswer in todayScrumAnswers)
             {
-                if (todayScrumAnswer.Question.QuestionStatement.Equals(_stringConstant.ScrumFirstQuestion))
+                if (todayScrumAnswer.Question.Type == 1 && todayScrumAnswer.Question.OrderNumber == 1)
                 {
-                    employeeScrumDetail.Answer1 = todayScrumAnswer.Answer.Split('\n');
+                    employeeScrumDetail.Answer1 = SplitAnswer(todayScrumAnswer.Answer);
                 }
-                if (todayScrumAnswer.Question.QuestionStatement.Equals(_stringConstant.ScrumSecondQuestion))
+                if (todayScrumAnswer.Question.Type == 1 && todayScrumAnswer.Question.OrderNumber == 2)
                 {
-                    employeeScrumDetail.Answer2 = todayScrumAnswer.Answer.Split('\n');
+                    employeeScrumDetail.Answer2 = SplitAnswer(todayScrumAnswer.Answer);
                 }
-                if (todayScrumAnswer.Question.QuestionStatement.Equals(_stringConstant.ScrumThirdQuestion))
+                if (todayScrumAnswer.Question.Type == 1 && todayScrumAnswer.Question.OrderNumber == 3)
                 {
-                    employeeScrumDetail.Answer3 = todayScrumAnswer.Answer.Split('\n');
+                    employeeScrumDetail.Answer3 = SplitAnswer(todayScrumAnswer.Answer);
                 }
             }
             return employeeScrumDetail;
         }
 
+        /// <summary>
+        /// Method to split the multiple scrum answers into an array
+        /// </summary>
+        /// <param name="Answer"></param>
+        /// <returns>scrum answer</returns>
+        private string[] SplitAnswer(string Answer)
```

Comment:
This method is created to split the scrum answers that are stored in the form of a single string into an array of strings. This splitting of answers is used at several places, so to reduce redundancy I created a separate method for it.